### PR TITLE
Update layer.py

### PR DESCRIPTION
**Pull request by niranjandasMM**
This pull request fixes the code from @AlessandroSaviolo repo named CNN from scratch , where an error occured in layer.py , i have replaced that code . the error was in a function named ' lr schedule ' where there was no conditon passed if the iteration is above than zero or below than 10,000 | this updated code in layer.py resolves the issue . 

You see , in layer.py file , the lr_schedule functon returns only learning rate when Iteration is 0 or above 10,000 , I think @AlessandroSaviolo forget to mention ' above zero or less than 10,000 ' should return learning rate also .Here is the working code ,replace it with that 'lr schedule function' in layer.py |

 ```
def lr_schedule(learning_rate, iteration):
    if iteration == 0:
        return learning_rate
    if (iteration >= 0) and (iteration <= 10000): # add this additional condition to resolve the bug
        return learning_rate
    if iteration > 10000:
        return learning_rate * 0.1
    if iteration > 30000:
        return learning_rate * 0.1
``` 
The learning rate is None because no argument was passed when the iteration is greater than 0 , So when the iteration 2nd hit , it became None and couldn't multipy with the Float value ... as @Hommus mentioned the error .... When you add an additional 'if condition' for greater than zero , it will no longer gives that error and the CNN will work completely fine. I hope my answer is clear ...

### CODE REVIEW

**Comment by niranjandasMM** [comment]
Hey Alessandro , your code is beautiful and super, mnist dataset work like
fire. But when I tried to use cifar10 , Converting it into Grey scale ,
then , its passing an error , showing in Pooling layer, but that is not the
actual error , it was causing because of massive weights maybe , which is
causing  values to be zero (Runtime warning error will be shown when u run
it ) and eventually at 2nd or 3rd interation it changes to Nan. Can u just
reflect and check,  in my research,  I changed the model, I externally
added relu layer rather than inside conv layer , don't know what's the
difference,  but mine is working but it's learning if very poor. I tried
Tanh in between layers , it learns but after sometimes it gets stuck .. Any
lead ?
